### ISSUE TITLE
Error handling for unparsable load input

### ISSUE THREAD

**Issue by WilcoFiers** [w3c/wcag-em-report-tool#248]
Right now, the tool just fails when it is requested to parse a JSON file that's invalid. It should properly notify users about this problem.

**Comment by rvantonisse** [comment]
Add a feedback message for opening wrong formatted json reports.
Stay in "open"-view.

**Comment by nitedog** [comment]
I believe @rvantonisse and @mhansma96 added such notifications in [version 3](https://github.com/w3c/wai-wcag-em-report-tool).

**Comment by hidde** [comment]
Correct, @nitedog!  Good news, [3.0.0 version of the WCAG-EM Report Tool](https://w3.org/WAI/eval/report-tool) is now live with notifications, so I am closing this issue.
